hobby-blog.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Manu Ponce Portfolio Site</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles/styles-sheet.css" />
+</head>
+
+<body>
+    <header>
+        <nav class="nav-wrapper">
+            <section class="left-side">
+                <section class="nav-link-wrapper">
+                    <a href="index.html">Home</a>
+                </section>
+
+                <section class="nav-link-wrapper">
+                    <a href="index.html">About</a>
+                </section>
+
+                <section class="nav-link-wrapper">
+                    <a href="index.html">Portfolio</a>
+                </section>
+
+                <section class="nav-link-wrapper">
+                    <a href="index.html">Blog</a>
+                </section>
+            </section>
+        </nav>
+    </header>
+
+    <section class="box-body">
+        <h1 class="title-name">Blog</h1>
+    </section>
+
+    <div class="blog-title">
+        <h2>Makakū</h2>
+    </div>
+
+    <div class="card">
+        <h2>Makakū</h2>
+        <h5>Seattle,WA, March 1, 2021</h5>
+        <div class="blog-img" width="20" height="30">
+            <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQ8-zqO0pluuvfXX_15S6ePZR1-UMqwRrKb8A&usqp=CAU"
+                alt="Italian Trulli">
+        <footer>
+        <p>In my current season, I feel this is my mountain I need to continue to climb and overcome.</p>
+    </footer>
+
+    <div class="blog-title">
+        <h2>Mauna Nui Ko`u</h2>
+    </div>
+
+    <div class="card">
+        <h2>My Great Mountain</h2>
+        <h5>Seattle,WA, April 1, 2021</h5>
+        <div class="blog-img" width="20" height="30">Ko`olau Mountains
+            <img src="https://onlyinhawaii.org/wp-content/uploads/2013/03/Hoomaluhia-Botanical-Garden-Koolau-mountain-range-300x191.jpg"
+                alt="Italian Trulli">
+        </footer>
```

Comment:
You still have open tags (the `tag` for instance) inside the `footer` which are still open when the browser hits this close tag.